### PR TITLE
WIP: Doc cleanup

### PR DESCRIPTION
**Pull request by omgaz**
Basically an entire re-write of Main.jsx and ALL examples.

Big thanks to @nimishjha :D

### NOTES
`npm run scaffold ComponentName` will set you up with everything you need to create a new component including an example file, don't forget to include in `docs/Layout/index.jsx` including the navigation. Speak to the UX team if you're not sure where to group something.

*Please make sure all examples are realistic and thoughtful and show workflow, interaction and usage.*

screen shot 2017-09-26 at 00 07 53

### CODE REVIEW

File: docs/new-examples/AccordionExample.jsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      ]
+};
+
+class AccordionExample extends React.Component {
+  constructor() {
+    super();
+    this.state = initialState;
+    this.toggleAccordionPanel = this.toggleAccordionPanel.bind(this);
+  }
+  toggleAccordionPanel(panelId) {
```
Comment (omgaz) [inline docs/new-examples/AccordionExample.jsx]: For readability and consistency with the team we prefer line breaks between methods.

File: docs/new-examples/BorderedWellExample.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          <Empty
+            collection={[]}
+            text="Empty"
+            svgSymbol={{ href: '/assets/svg-symbols.svg#checklist-incomplete' }}
+          />
+          <PageTitle isFooter>
+            <SvgSymbol />
+          </PageTitle>
+        </BorderedWell>
+      );
```
Comment (omgaz) [inline docs/new-examples/BorderedWellExample.jsx]: The indentation seems off here.

Comment (nimishjha) [reply docs/new-examples/BorderedWellExample.jsx]: Yeah, fixing. I usually use tabs so am getting thrown off by the 2-space indentation.

File: docs/new-examples/AccordionExample.jsx (hunk 1 above)
Comment (nimishjha) [reply docs/new-examples/AccordionExample.jsx]: Fixing.

File: config/dev.js [hunk 3]
```diff
@@ -55,6 +55,8 @@ module.exports = merge(commonConfig, {
     publicPath, // match the output `publicPath`
     port: PORT,
     host: HOST,
+    disableHostCheck: true,
+    disableHostCheck: true,
```
Comment (omgaz) [inline config/dev.js]: twice

File: docs/examples/AlertExample.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+class AlertExample extends React.PureComponent {
+  render() {
+    return (<Alert type="danger">Error: Unable to save. We can't talk to the API, try again in a bit.</Alert>);
+  }
+}
+
+
+const exampleProps = {
+  componentName: 'Alert',
+  notes: '',
```
Comment (omgaz) [inline docs/examples/AlertExample.jsx]: remove empties

File: docs/index.html [hunk 5]
```diff
@@ -5,12 +5,15 @@
   <title>Adslot UI</title>
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <meta name="description" content="Core Component Library. By Adslot">
-  <link rel='stylesheet' href='https://fonts.googleapis.com/css?family=Roboto:300,700' />
+  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,700" />
   <link rel='stylesheet' href='assets/adslot-ui-main.css' />
+  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/tomorrow.min.css">
```
Comment (omgaz) [inline docs/index.html]: remove

File: docs/index.html [hunk 6]
```diff
@@ -5,12 +5,15 @@
...
 </head>
 <body>
   <div id="app">APPLICATION CONTENT</div>
 
   <script type="text/javascript" src="assets/app.js"></script>
+  <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js"></script>
+  <script>hljs.initHighlightingOnLoad();</script>
```
Comment (omgaz) [inline docs/index.html]: remove

**Comment by lteacher** [comment]
LGTM - Merging

**Pull request merged**